dolphinscheduler-dao/src/main/resources/sql/upgrade/3.2.0_schema/postgresql/dolphinscheduler_ddl.sql

```diff
@@ -118,4 +118,7 @@ $$ LANGUAGE plpgsql;
 d//
 delimiter ;
 select uc_dolphin_T_t_ds_task_instance_R_test_flag();
-DROP FUNCTION uc_dolphin_T_t_ds_task_instance_R_test_flag();
\ No newline at end of file
+DROP FUNCTION uc_dolphin_T_t_ds_task_instance_R_test_flag();
+
+
+create UNIQUE index IF NOT EXISTS idx_process_definition_log_code_version on t_ds_process_definition_log (code,version);
```

Comment:
tips, do not forget to change the unique index name when we change it in `dolphinscheduler_postgresql.sql`